src/script.js

```diff
@@ -222,6 +222,25 @@ export function getLocale() : LocaleType {
         if (COUNTRY_LANGS.hasOwnProperty(country) && COUNTRY_LANGS[country].indexOf(lang) !== -1) {
             // $FlowFixMe
             return { country, lang };
+        } else {
+            // We will infer country from language if there is only one possible country match
+            const possibleLocales = [];
+            for (const possibleCountry of Object.keys(COUNTRY_LANGS)) {
+                const languages = COUNTRY_LANGS[possibleCountry];
+                const language = languages.find(x => x === lang);
+
+                if (language) {
+                    possibleLocales.push({ country: possibleCountry, lang: language });
```

Comment:
Could we simplify this by just doing a `return` here, rather than building an array and breaking out of the loop?